docs/network/evm-nodes.md

```diff
@@ -17,6 +17,22 @@ You can get the kernel by importing it from a running Etherlink Smart Rollup nod
 
 Running an Etherlink EVM node on Etherlink Mainnet requires a computer with 500GB of disk space and at least 16GB RAM.
 
+## Modes
+
+The EVM node supports these history modes:
+
+- `archive` (the default): The node stores a copy of all available Etherlink information.
+
+- `full` (available starting with version 0.17): The node stores all of the necessary information to construct the current Etherlink state plus the states for a certain number of previous days, known as the _retention period_.
+
+- `rolling` (available starting with version 0.16): The node stores the current context plus the complete transaction data for a certain number of previous days, known as the _retention period_.
+The default `rolling` mode stores the complete data for the previous 14 days.
```

Comment:
We don’t have a default retention period anymore. We decided it was probably better to let the users define them. The octez node has a default retention period that has been changed a few times and the resulting experience is not ideal IMO, compared to push users to ask themselves how much day they need.

Reply:
Removed the default for rolling mode.